draft-ietf-rats-msg-wrap.md

```diff
@@ -274,6 +274,13 @@ Since the collection type is recursive, implementations may limit the allowed de
 
 Although initially designed for the composite Attester use case, the CMW collection can be repurposed for other use cases requiring CMW aggregation.
 
+### CMW Collections' role in composite Attester topology
+
+A CMW Collection's tree structure is not required to be a spanning tree of the a system's composite Attester topology, i.e, it need not be a reflection of domain membership nor domain dependency.
```

Comment:
I am not sure a reader of this document necessarily understands what "domain membership" and "domain dependency" mean.

Reply:
What if we delete everything after the comma starting with "i.e."?

Reply:
> What if we delete everything after the comma starting with "i.e."?

WFM